src/cpp/shared_core/json/Json.cpp

```diff
@@ -630,6 +632,63 @@ Error Value::validate(const std::string& in_schema) const
    return Success();
 }
 
+Error Value::coerce(const std::string& in_schema,
```

Comment:
Nit: Could you move this definition to match the position in the hpp?

Reply:
Will do.